packages/core/src/BaseCheckBox/BaseCheckBox.styles.ts

```diff
@@ -13,14 +13,15 @@ export const { staticClasses, useClasses } = createClasses("HvBaseCheckBox", {
     cursor: "pointer",
     "&:hover": {
       backgroundColor: theme.colors.bgHover,
+      borderRadius: theme.radii.round,
     },
 
     "& svg": {
       width: 16,
       height: 16,
       color: theme.colors.bgContainer,
-      borderRadius: theme.radii.none,
-      border: `1px solid ${theme.colors.text}`,
+      borderRadius: theme.radii.base,
```

Comment:
unsure if it's related to this change, but the new radius is showing a duplicated & rounded error border.

![image](https://github.com/user-attachments/assets/c15187bc-c85d-4ada-a630-ed06d331596c)

Reply:
Yeah, we have two "invalid state" lines in the component. One for when we only have the checkbox and the other to the container (that includes) the label. When we have a label is just a matter of removing the line from the checkbox (the container already has it hence the double border). But when we only have the checkbox the case is trickier. I went for a `::after` pseudo-class and I think it works. Let me know what you think.